test/test_affine.jl

```diff
@@ -417,4 +417,36 @@
             @test all(abs.(p.constraints[1].dual - dual) .<= TOL)
         end
     end
+
```

Comment:
Could we add the test for `permutedims_matrix` from here: https://github.com/ericphanson/QuantumSDPs.jl/blob/97225a7616f08f46de568091dcf63956b2125b1c/test/runtests.jl#L18-L28? It should work by just changing `QuantumSDPs.permuteddims_matrix` to `Convex.permuteddims_matrix`.

Reply:
done.